src/Sylius/Bundle/AdminBundle/Resources/views/Layout/_channelLinks.html.twig

```diff
@@ -1,3 +1,5 @@
+{% set homepagePath = sylius_bundle_loaded_checker('SyliusShopBundle') ? path('sylius_shop_homepage') : '/' %}
```

Comment:
```suggestion
{% set homepage_path = sylius_bundle_loaded_checker('SyliusShopBundle') ? path('sylius_shop_homepage') : '/' %}
```
according to our [Conventions](https://docs.sylius.com/en/1.12/book/contributing/code/conventions.html)